### ISSUE TITLE
Describe why the root view "specialness" aren't just browser bugs

### ISSUE THREAD

**Issue by RByers** [bokand/NonDocumentRootScroller#2]
The main objection I imagine to this proposal is: "All the reasons you list for how the root scroller is special are just browser bugs - rather than introduce a new API and hoist the problem on developers, browsers should just fix those bugs".

IIRC you had some compelling arguments against this, but I don't see them in the explainer.  Eg. there's an accessibility argument around not wanting to depend on heuristics to identify the primary scroller when an accessibility tool (or just the spacebar) is used to "scroll the page".

I think there was also a predictability argument: helping developers reason about exactly when scrolling a given div will hide the top controls.  eg. it's better to have an API that can fail with a specific exception for why a div can't be a root scroller, then to silently change behavior when there's a subtle issue like a single-pixel gap or occlusion etc.

**Comment by bokand** [comment]
Yeah, I'll try to compile some more thorough arguments - though allowing the URL bar to show/hide isn't a browser bug but innate in how the web works today. That alone is probably compelling to many.

**Comment by RByers** [comment]
>  though allowing the URL bar to show/hide isn't a browser bug but innate in how the web works today.

I meant that the fact that the URL bar hiding is coupled to the root scroller instead of, for example, to any full page scroller.  I think that's what people could argue was a browser bug.

**Comment by bokand** [comment]
Ah, got it. I'll give this some thought and update the explainer.
